### ISSUE TITLE
Fresh install includes buggy brq

### ISSUE THREAD

**Issue by tynes** [bcoin-org/bmultisig#17]
The error received is:

```
TypeError: Cannot read property 'buffer' of null
    at RequestOptions.toHTTP (request-browser.js?8340:290)
    at _request (request-browser.js?8340:369)
    at request (request-browser.js?8340:409)
    at MultisigClient.request (client.js?9aa0:174)
    at MultisigClient.get (client.js?9aa0:223)
    at MultisigClient.getWallets (client.js?9995:87)
```

To reproduce:

```
$ mkdir /tmp/multisig-debug && cd /tmp/multisig-debug
$ npm init -y >/dev/null
$ npm install bmultisig
$ grep -rin tohttp node_modules | grep browser

> node_modules/brq/lib/request-browser.js:286:  toHTTP() {
> node_modules/brq/lib/request-browser.js:369:  const response = await fetch(opt.toURL(), opt.toHTTP());

$ less +286 node_modules/brq/lib/request-browser.js

  toHTTP() {
    return {
      method: this.method,
      headers: this.getHeaders(),
      body: this.body.buffer,  // bug here
      mode: 'cors',
      credentials: 'include',
      cache: 'no-cache',
      redirect: 'follow',
      referrer: 'no-referrer'
    };
  }
```
Digging a little deeper:

```
$ cd /tmp/multisig-debug/node_modules/bmultisig
$ jq .dependencies < package.json
{
  "bclient": "~0.0.2",
  "bcoin": "github:bcoin-org/bcoin#e093b2b",
  "bcrypto": "~0.2.0",
  "bdb": "~0.1.0",
  "bevent": "0.0.2",
  "blgr": "~0.0.2",
  "bmutex": "~0.0.2",
  "bstring": "~0.0.2",
  "bufio": "~0.1.0",
  "bval": "~0.0.2",
  "bweb": "~0.0.2"
}
```

I think that the problem lies in an old version of `bclient` being imported, you can see we are at version `0.1.3` on npm here - https://www.npmjs.com/package/bclient

**Comment by tynes** [comment]
In the `package.json`, you can see that it includes the newer version of `bclient`
https://github.com/bcoin-org/bmultisig/blob/master/package.json#L32

I think we need to publish the latest version of this to `npm`

**Comment by nodar-chkuaselidze** [comment]
Published `v0.0.3` with bug fixes and updated dependencies.
